dev-cpp/string-theory/string-theory-3.7.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2022 Gentoo Authors
```

Comment:
Use our commit-helper tools such as **pkgdev** to automate this :)

Reply:
test will now trigger dependency on dev-cpp/gtest.